frontstage/error_handlers.py

```diff
@@ -43,7 +36,14 @@ def connection_error(error):
 
 
 @app.errorhandler(JWTValidationError)
-def connection_error_jwt_validation(error):  # pylint: disable=unused-argument
+def jwt_validation_error(error):  # pylint: disable=unused-argument
     return redirect(url_for('error_bp.not_logged_in_error_page',
                             _external=True,
                             _scheme=getenv('SCHEME', 'http')))
+
+
+@app.errorhandler(Exception)
+def server_error(error):  # pylint: disable=unused-argument
```

Comment:
Why is the error not logged out? I noticed this with most of the repos just wondering if there is a reason the error itself isn't logged?